src/Bundle/Tests/DependencyInjection/Compiler/PagerfantaBridgePassTest.php

```diff
@@ -26,7 +25,7 @@ final class PagerfantaBridgePassTest extends AbstractCompilerPassTestCase
      */
     public function it_creates_aliased_services_and_changes_the_view_factory_class(): void
     {
-        $this->registerService('pagerfanta.twig_extension', PagerfantaExtension::class);
+        $this->registerService('pagerfanta.twig_extension', class_exists('Pagerfanta\Twig\Extension\PagerfantaExtension') ? 'Pagerfanta\Twig\Extension\PagerfantaExtension' : 'BabDev\PagerfantaBundle\Twig\PagerfantaExtension');
```

Comment:
You can always use `Pagerfanta\Twig\Extension\PagerfantaExtension` here.  Since the bundle's 2.5 release, that's the class used for the `pagerfanta.twig_extension` service, and with the version constraints given, it'll always be available.